trite-osprey/backend/db.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs')
+
+if(fs.existsSync('.env') ){
```

Comment:
this should be conditional to your `NODE_ENV` and not based on the existence of the file. environment variables are rarely optional